zipline/assets/continuous_futures.pyx

```diff
@@ -37,6 +39,21 @@ import warnings
 from zipline.utils.calendars import get_calendar
 
 
+def delivery_predicate(codes, contract):
+    # This relies on symbols that are construct following a pattern of
+    # root symbol + delivery code + year, e.g. PLF16
+    # This check would be more robust if the future contract class had
+    # a 'delivery_month' member.
+    delivery_code = contract.symbol[2]
```

Comment:
Is the `delivery_code` guaranteed to be the third character of the symbol, i.e. the root symbol is always 2 characters?